src/applications/MachineShopSimulator.java

```diff
@@ -86,22 +89,30 @@ private void setUpJobs(SimulationSpecification specification) {
         // input the jobs
         Job theJob;
         for (int i = 1; i <= specification.getNumJobs(); i++) {
-            int tasks = specification.getJobSpecifications(i).getNumTasks();
+            int tasks = Task.getNumTasks(specification, i);

```

Comment:
I'm not sure I understand why you're moving from methods attached to object (`specification.getJobSpecifications(i)`) to static methods (`Task.getNumTasks(…)`). I'm not sure I _object_, but I'm definitely not sure where you're headed with this and would love to hear more.